common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -191,8 +191,13 @@ def test_enroll(self, course_modes, enrollment_mode):
             )
 
         # Create an enrollment
-        self.assert_enrollment_status()
+        resp = self.assert_enrollment_status()
```

Comment:
originally I did that, but many of the other tests use `resp`, so I thought it'd be better to be consistent with everything else.